### PR TITLE
[RA1 Ch4] More precise DPDK usage

### LINKED ISSUE #1388: [RA1 Ch4] 4.2.2.5 More precise on DPDK usage
I checked with original member of DPDK team and we have more precise proposal for DPDK usage text in [4.2.2.5. Compute Nodes](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#4225-compute-nodes), separating what goes into Cloud infra vs Workload:

| Layer | Description |
|-------------|--|
| Cloud infrastructure | Important is placement of NICs to get to balanced IO system, and configuration of NIC features. Server BIOS and Host OS kernel command line settings are described on DPDK.org in release notes and performance reports. Disabling power settings (like Intel Turbo Boost Technology) brings stable performance results, although understanding if and when they benefit workloads and enabling them can achieve better performance results. |
| Workload | DPDK uses core affinity along with 1G or 2M Huge Pages, NUMA settings, and DPDK Poll Mode Drivers (PMD) to get the best performance. DPDK versions xx.11 are Long-Term Support maintained stable release with back-ported bug fixes for a two-year period. |

### PR DESCRIPTION
**Pull request by petorre**
Fixes #1388.

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter04.md [hunk 1]
```diff
@@ -143,8 +143,14 @@ CPU reservation for host: 1 core per NUMA
...
 -	HW specifications
     -	Boot disks are dedicated with Flash technology disks
-    - In case of DPDK usage, cores (together with their sibling threads) must be reserved for DPDK Poll Mode Drivers (PMD), hugepages set to at least 1 GB, and all power settings disabled in the BIOS. The number of cores reserved for this overhead depends on the desired throughput (the number of Mpps we want to deliver).
-    
+
+- In case of DPDK usage:
+
+| Layer | Description |
+|-------------|--|
+| Cloud infrastructure | Important is placement of NICs to get to balanced IO system, and configuration of NIC features. Server BIOS and Host OS kernel command line settings are described on DPDK.org in release notes and performance reports. Disabling power settings (like Intel Turbo Boost Technology) brings stable performance results, although understanding if and when they benefit workloads and enabling them can achieve better performance results. |
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter04.md]: @petorre "Important is placement of NICs to get to balanced IO system .." is this asking for a suitable NIC per NUMA? If so can we clarify? Thanks

**Comment by pgoyal01** [comment]
@petorre Thanks for getting this important content. See just one question relative to the "balanced IO" above.

File: doc/ref_arch/openstack/chapters/chapter04.md (hunk 1 above)
Comment (petorre) [reply doc/ref_arch/openstack/chapters/chapter04.md]: Yes. Just for whatever reason it did not allow me to Edit it here, so is now in #1402 . (Sorry if I don't do it right)

**Comment by petorre** [comment]
> @petorre Thanks for getting this important content. See just one question relative to the "balanced IO" above.

In #1402.

**Comment by ASawwaf** [comment]
@petorre , thanks for share input, but there is no numbers of Core for DPDK usages , did you figure it out

File: doc/ref_arch/openstack/chapters/chapter04.md (hunk 1 above)
Comment (CsatariGergely) [inline doc/ref_arch/openstack/chapters/chapter04.md]: Can you please add links to the release notes and performance reports in DPDK.org?

File: doc/ref_arch/openstack/chapters/chapter04.md [hunk 2]
```diff
@@ -143,8 +143,14 @@ CPU reservation for host: 1 core per NUMA
...
+| Workload | DPDK uses core affinity along with 1G or 2M Huge Pages, NUMA settings, and DPDK Poll Mode Drivers (PMD) to get the best performance. DPDK versions xx.11 are Long-Term Support maintained stable release with back-ported bug fixes for a two-year period. |
```
Comment (DheerajDang) [inline doc/ref_arch/openstack/chapters/chapter04.md]: to get best performance, do we need multiple PMD threads at compute host level?

File: doc/ref_arch/openstack/chapters/chapter04.md (hunk 1 above)
Comment (petorre) [reply doc/ref_arch/openstack/chapters/chapter04.md]: Done in #1402. (I cannot Edit this one, will do here if that changes)

**Comment by petorre** [comment]
(Now I can again edit this PR so will close #1402 ).

File: doc/ref_arch/openstack/chapters/chapter04.md (hunk 2 above)
Comment (petorre) [reply doc/ref_arch/openstack/chapters/chapter04.md]: It depends on design details explained in DPDK Programming Guide [PMD](https://doc.dpdk.org/guides/prog_guide/poll_mode_drv.html) that I would not add in RA text.

**Comment by petorre** [comment]
> @petorre , thanks for share input, but there is no numbers of Core for DPDK usages , did you figure it out

Added "on reserved cores".

**Comment by rabi-abdel** [comment]
@CsatariGergely can you please re-review/approve

**Pull request merged**